websockets/bridge.go

```diff
@@ -0,0 +1,116 @@
+package websockets
+
+import (
+	"fmt"
+
+	"github.com/gorilla/websocket"
+	"github.com/pokt-network/poktroll/pkg/polylog"
+)
+
+// bridge routes data between an Endpoint and a Client. One bridge represents
+// a single WebSocket connection between a Client and a WebSocket Endpoint.
+//
+// Full data flow: Client <------> PATH <------> WebSocket Endpoint
+type bridge struct {
+	logger       polylog.Logger
+	endpointConn *connection
```

Comment:
Something similar to `Full data flow: Client <------> PATH <------> WebSocket Endpoint` for endpoint/client conn would be great.